optimizely/helpers/sdk_settings.py

```diff
@@ -47,8 +48,11 @@ def __init__(
             `fetch_qualified_segments(user_key, user_value, options)`.
           odp_event_manager: A custom odp event manager. Required method is:
             `send_event(type:, action:, identifiers:, data:)`
-          fetch_segments_timeout: A fetch segment timeout in seconds (optional).
-          odp_event_timeout: A send odp event timeout in seconds (optional).
+          odp_segment_request_timeout: A fetch segment timeout in seconds (optional) - how long to wait for
+            odp platform to respond before failing.
+          odp_event_request_timeout: A send odp event timeout in seconds (optional) - how long to wait for
+            odp platform to respond before failing.
```

Comment:
First half of these comments doesn't add much- let's rephrase these descriptions to something like:
```suggestion
          odp_segment_request_timeout: Time to wait in seconds for fetch_qualified_segments request to send successfully (optional)
          odp_event_request_timeout: Time to wait in seconds for send_odp_events request to send successfully
```